packages/core/src/os.ts

```diff
@@ -182,7 +182,11 @@ export function launch(
     );
 
     // Check first if we need to add logging
+    // Run in a new process group to prevent child processes from being
+    // terminated when signals are sent to the parent's process group.
+    // See: https://github.com/endorhq/rover/issues/374
     let newOpts: Options = {
+      detached: true,
       ...expandedOptions,
     } as Options;
```

Comment:
Let's keep this approach for now. We might encounter some issues when running processes in the host (for example, to expand a task metadata). We can change this behavior to a custom flag after we switch to ACP, as MCP configuration is applied directly.

Reply:
Sounds good!